src/client/eppo-client.ts

```diff
@@ -351,6 +358,90 @@ export default class EppoClient implements IEppoClient {
     );
   }
 
+  public getBanditAction(
```

Comment:
nit: add jsdocs

Reply:
That is a good reminder to add this to the interface, where we have our JS docs. Of course, Greg is axing that so it will probably end up back here on rebase 😅